.github/tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+      
+    - name: Setup Gradle
```

Comment:
The `gradlew` is meant for direct running, and as a result, already has the permissions set. We can remove this step.

Reply:
Thanks for the nit, It has become a habit as I work on windows haha. 
Sending Updated commit now :)